From 58f6fccc040beb835d76e5b97a61c44fa815f31f Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Wed, 14 Nov 2018 16:53:47 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399pro-npu: Add
 dynamic-power-coefficient for cpu

The average value of dynamic-power-coefficient is about 74.

Change-Id: Id80521440e0cbdb677344bf5becf092e4311c499
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi b/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
index c8cc49a269df..04d8b075f776 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
@@ -32,6 +32,7 @@
 			enable-method = "psci";
 			clocks = <&cru ARMCLK>;
 			operating-points-v2 = <&cpu0_opp_table>;
+			dynamic-power-coefficient = <74>;
 		};
 
 		cpu1: cpu@1 {
@@ -41,6 +42,7 @@
 			enable-method = "psci";
 			clocks = <&cru ARMCLK>;
 			operating-points-v2 = <&cpu0_opp_table>;
+			dynamic-power-coefficient = <74>;
 		};
 	};
 
-- 
2.35.3

